Demo/Application/Features/PayPal - Native Checkout/BraintreeDemoPayPalNativeCheckoutViewController.swift

```diff
@@ -58,16 +66,34 @@ class BraintreeDemoPayPalNativeCheckoutViewController: BraintreeDemoPaymentButto
         }
     }
 
-    @objc func tappedPayPalVault(_ sender: UIButton) {
-        progressBlock("Tapped PayPal - Vault using BTPayPalNativeCheckout")
-        sender.setTitle("Processing...", for: .disabled)
-        sender.isEnabled = false
-
-        let request = BTPayPalNativeVaultRequest()
-
-        payPalNativeCheckoutClient.tokenize(request) { nonce, error in
-            sender.isEnabled = true
+        @objc func tappedVaultCheckout(_ sender: UIButton) {
+            progressBlock("Tapped PayPal - Vault using BTPayPalNativeCheckout")
+            sender.setTitle("Processing...", for: .disabled)
+            sender.isEnabled = false
+            
+            let request = BTPayPalNativeVaultRequest()
+            
+            payPalNativeCheckoutClient.tokenize(request) { nonce, error in
+                sender.isEnabled = true
+              
+                guard let nonce else {
+                  self.progressBlock(error?.localizedDescription)
+                  return
+                }
+                self.completionBlock(nonce)
+          }
+      }
 
+        @objc func tappedCheckoutWithVault(_ sender: UIButton) {
```

Comment:
Nitpick - looks like the indentation on this func and the one above got messed up a bit!

Reply:
@scannillo Fixed!

Reply:
Looks like the functions that start on line 87 and 69 are 4 spaces too far to the right.

I know NXO uses 2 spaces and BT/PPCP use 4. So updating your Xcode text editing preferences temporarily and then doing a `CTRL + i` might be the easiest: 


<img width="815" alt="Screenshot 2023-08-15 at 6 14 34 PM" src="https://github.com/braintree/braintree_ios/assets/35243507/78a824ed-958b-4ec9-ae48-43c89450e935">

Reply:
Thanks @scannillo, I did your recommendation and the indentations should be fixed now